### ISSUE TITLE
`Copy OC login command` not working as `oc` is not in the PATH

### ISSUE THREAD

**Issue by guillaumerose** [code-ready/tray-macos#64]
The copy button works well but the user doesn't have oc in his PATH. The command fails directly.

crc oc-env is missing or we should install oc in the PATH.

**Comment by anjannath** [comment]
Yeah, we can improve the experience by adding the `oc-env` part also in the copied command. If installed from the package do we put the `oc` in PATH directly?
